### ISSUE TITLE
Error when trying to run segmentation

### ISSUE THREAD

**Issue by edelaserna** [MouseLand/cellpose-napari#20]
I recently installed napari and the cellpose napari plugin. I am able to open napari, load my image stack, and then open the cellpose plugin to input segmentation settings before clicking "run segmentation". However, when I click on "compute the diameter from the image" or "run segmentation", I get the same error:

```
ImportError: cannot import name 'logger' from 'cellpose' (/Users/username/anaconda2/envs/napari-env/lib/python3.8/site-packages/cellpose/__init__.py)

```
I installed the python package of napari as recommended using conda. I installed the napari plugin using the napari GUI installer. I've deleted the napari conda environment and recreated it 3 times now and still get the same error. The first time I used pip to install napari, second time I tried making the environment with python 3.9 instead of python 3.8, third time I used conda to install napari. 

Final attempt was to uninstall the cellpose plugin using the napari GUI installer, and then install cellpose via terminal with pip. Still get the same error.

I'm not quite sure what else I can try to resolve this issue. I don't fully understand the nature of the error either, but assumed it was related to an unsuccessful install. Any help would be greatly appreciated! Thanks!

**Comment by tlambert03** [comment]
yeah, I noticed this today as well... I think cellpose has changed in the meantime, and this plugin needs some updating.  based on the git history, I think `pip install cellpose==0.6.5` will work for you in the meantime

**Comment by edelaserna** [comment]
Ah okay. Thanks for the sanity check!

**Comment by dispiche** [comment]
Hello I have been following this tutorial: https://alisterburt.github.io/napari-workshops/SwissBIAS-1121/intro.html
I have encountered the same issue when trying to calculate a diameter with the cellpose-napari plugin. 
For me running "pip install cellpose==0.7.2" seems to fix the issue

This is the Error I recieved: 

Traceback (most recent call last):
  File "D:\miniforge3\envs\napari-tutorial\lib\site-packages\magicgui\widgets\_bases\value_widget.py", line 57, in _on_value_change
    self.changed.emit(value)
  File "psygnal\_signal.py", line 670, in psygnal._signal.SignalInstance.emit
  File "D:\miniforge3\envs\napari-tutorial\lib\site-packages\magicgui\events.py", line 85, in _run_emit_loop
    cb(*args[:max_args])
  File "D:\miniforge3\envs\napari-tutorial\lib\site-packages\cellpose_napari\_dock_widget.py", line 327, in _compute_diameter_shape
    _report_diameter(diam)
  File "D:\miniforge3\envs\napari-tutorial\lib\site-packages\cellpose_napari\_dock_widget.py", line 296, in _report_diameter
    from cellpose import logger
ImportError: cannot import name 'logger' from 'cellpose' (D:\miniforge3\envs\napari-tutorial\lib\site-packages\cellpose\__init__.py)

I installed the both cellpose and the cellpose-napari plugin using conda. 

My versions during the bug are as follows: 
Python
Python 3.9.9 | packaged by conda-forge | (main, Dec 20 2021, 02:40:06) [MSC v.1929 64 bit (AMD64)] on win32
cellpose
Version: 0.8.0
cellpose-napari
Version: 0.1.4
napari
Version: 0.4.13

I wanted to thank the developers for all their hard work for our community and if there is any more info i can give to help please drop me a message

**Comment by tlambert03** [comment]
fixed in #29, in newest release
